nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/main/java/org/apache/nifi/processors/mqtt/ConsumeMQTT.java

```diff
@@ -288,9 +289,13 @@ public void process(final OutputStream out) throws IOException {
 
             String transitUri = new StringBuilder(broker).append(mqttMessage.getTopic()).toString();
             session.getProvenanceReporter().receive(messageFlowfile, transitUri);
+            String uuid = messageFlowfile.getAttribute(CoreAttributes.UUID.key());
```

Comment:
A bit of a nit pick, this is only referenced in the logger statement below so it should be created in there, not outside of it.